administrator/components/com_fields/tmpl/fields/default.php

```diff
@@ -179,7 +179,7 @@
                                         <?php echo $this->escape($item->type); ?>
                                     </td>
                                     <td>
-                                        <?php echo $this->escape($item->group_title); ?>
+                                        <?php echo !empty($item->group_title) ? $this->escape($item->group_title) : Text::_('JNONE'); ?>
```

Comment:
```suggestion
                                        <?php echo !empty($item->group_title) ? $this->escape($item->group_title) : '[ ' . Text::_('JNONE') . ' ]'; ?>
```
Someone could name a field group "None". To distinguish that, the list view should show "[ None ]" like PR #45232 does it e.g. here: https://github.com/joomla/joomla-cms/pull/45232/files#diff-bd22b34aa4cc58086d9e4756bbdb29628f0950b4723c592ef5ebc6f57bdf92dcR329